src/Lesson1/Game.java

```diff
@@ -0,0 +1,37 @@
+package Lesson1;
+
+public class Game {
+    private Obstacles[] obstacles;
+    private Participant[] participants;
+
+    public void setObstacles(Man man, Robot robot, Cat cat, Obstacles... obstacles){
```

Comment:
классы Man, Robot, Cat являются реализациями интерфейса Participant, задавать их нужно в другом методе.
```suggestion
    public void setObstacles(Obstacles... obstacles){
```